go.mod

```diff
@@ -22,21 +22,22 @@ require (
 	github.com/r3labs/diff/v2 v2.15.1
 	github.com/stretchr/testify v1.8.1
 	github.com/tigera/api v0.0.0-20230406222214-ca74195900cb
-	go.uber.org/zap v1.24.0
+	go.uber.org/zap v1.23.0
 	golang.org/x/crypto v0.6.0
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.26.3
-	k8s.io/apiextensions-apiserver v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/client-go v0.26.3
-	k8s.io/kube-aggregator v0.26.3
-	k8s.io/utils v0.0.0-20230313181309-38a27ef9d749
-	sigs.k8s.io/controller-runtime v0.14.6
+	k8s.io/api v0.25.8
```

Comment:
There is now also:
```
	k8s.io/api v0.25.10
	k8s.io/apiextensions-apiserver v0.25.10
	k8s.io/apimachinery v0.25.10
	k8s.io/client-go v0.25.10
	k8s.io/kube-aggregator v0.25.10
```

Reply:
Lets do this in a follow up one, I don't want to hold this up any.